test/supplemental_genbank_tests/genbank_upload_full_test.py

```diff
@@ -167,3 +171,125 @@ def test_simple_shock_upload(self):
         self.assertIsNotNone(result['genome_ref'])
         self.assertTrue(int(
             result['genome_info'][10]['Number of Protein Encoding Genes']) > 0)
+
+    def _run_test_fail(self, params, error_message, mass=True):
+        with pytest.raises(Exception) as got:
+            if mass:
+                self.getImpl().genbanks_to_genomes(self.getContext(), params)
+            else:
+                self.getImpl().genbank_to_genome(self.getContext(), params)
+        assert_exception_correct(got.value, ValueError(error_message))
+
+    def _get_objects_data(self, refs):
+        results = self.getDfuClient().get_objects({'object_refs': refs})
+        return [result["info"] for result in results["data"]]
+
+    def test_genbank_to_genome_invalid_workspace(self):
+        genome_name = "GCF_000970165.1_ASM97016v1_genomic.gbff.gz"
+        params = {
+            'file': {'path': f"data/gbff/{genome_name}"},
+            'genome_name': genome_name,
+        }
+        self._run_test_fail(
+            params,
+            "Exactly one of a 'workspace_id' or a 'workspace_name' parameter must be provided",
+            False,
+        )
+
+    def test_genbanks_to_genomes(self):
+        genome_name1 = "GCF_000970165.1_ASM97016v1_genomic.gbff.gz"
+        genome_name2 = "GCF_000970185.1_ASM97018v1_genomic.gbff.gz"
+        results = self.getImpl().genbanks_to_genomes(
+            self.getContext(),
+            {
+                "workspace_id": self.getWsID(),
+                "inputs": [
+                    {
+                        "file": {"path": f"data/gbff/{genome_name1}"},
+                        "genome_name": genome_name1
+                    },
+                    {
+                        "file": {"path": f"data/gbff/{genome_name2}"},
+                        "genome_name": genome_name2
+                    }
+                ]
+            }
+        )[0]['results']
+
+        self.assertEqual(len(results), 2)
+        self.assertEqual(results[0]['genome_info'][1], genome_name1)
+        self.assertEqual(results[1]['genome_info'][1], genome_name2)
+
+        assembly_infos = [result['assembly_info'] for result in results]
+        genome_infos = [result['genome_info'] for result in results]
+
+        assembly_refs = [result['assembly_ref'] for result in results]
+        genome_refs = [result['genome_ref'] for result in results]
+
+        self.assertEqual(assembly_infos, self._get_objects_data(assembly_refs))
+        self.assertEqual(genome_infos, self._get_objects_data(genome_refs))
```

Comment:
This is just testing that the info returned from GFU, which GFU got from the workspace, is the same as what's in the workspace. It needs to test that the info is correct and as expected based on the uploaded data

Reply:
👍

Reply:
:+1: 

But see comments above about testing the data / prov / info for genomes and assemblies

Reply:
Will test the data/prov/info for assemblies in a future PR.